### ISSUE TITLE
All definitions should be indexed

### ISSUE THREAD

**Issue by burblebee** [cplusplus/draft#329]
It was decided at the Rap. mtg that all definitions should be indexed, i.e., definitions which use \term{} should be fixed to use \defn{} or \defnx{}.

**Comment by jensmaurer** [comment]
The fun part of this is that some definitions use \term and have an \indextext next to it. I agree \defnx is technically superior, though. For what it's worth, there are 684 mentions of \term, which should probably all become either \defn or \placeholder or something else.

**Comment by tkoeppe** [comment]
I think PR #980  addresses part of that. It's pending review.

**Comment by jensmaurer** [comment]
I would have been nice to mention #329 in PR #980 to avoid duplicate work...

**Comment by tkoeppe** [comment]
That's true. I find it useful to browser the pending pull requests periodically to see what's pending. Feel free to edit pending PRs with issue links as you find them.

**Comment by Eelis** [comment]
Update to quantify the "big" label: there's now 216 uses of \term left.

**Comment by jwakely** [comment]
As I mentioned in #3435, these look wrong:

````
declarations.tex:Except for this special case, a parameter shall not have type \term{cv}
declarations.tex:if any of its \term{namespace-definition}{s} is exported,
expressions.tex:\tcode{std::type_info} or \tcode{const} \term{name} where \term{name} is an
locales.tex:\term{explicit},
````

**Comment by AlisdairM** [comment]
Bump.  Is this something we should consider for C++26? or maybe an early project after the final C++26 doc is shipped to publish?

My quick `grep | wc` suggests we have made slow progress, down to 188 occurrences --- but some of those are in new wording!

I think there are some low-hanging fruit for obvious `\term` -> `\defn` changes, but the majority remaining are less clear to me.

Are we interesting in partial fixes of just the most obvious cases?
